src/logger.ts

```diff
@@ -16,11 +16,18 @@ export default function DefaultLogger (options?: Pino.LoggerOptions): Pino.Logge
       censor: '***'
     },
     // if in local environment use pretty print logs
-    ...process.env.NODE_ENV !== 'production' && isInstalled('pino-pretty') && {
-      prettyPrint: { translateTime: 'UTC:yyyy-mm-dd\'T\'HH:MM:ss.l\'Z\'' } // show timestamp instead of epoch time
+    // ...process.env.NODE_ENV !== 'production' && isInstalled('pino-pretty') && {
+    //   prettyPrint: { translateTime: 'UTC:yyyy-mm-dd\'T\'HH:MM:ss.l\'Z\'' } // show timestamp instead of epoch time
```

Comment:
(nit) do we need these commented lines commented anymore?

Reply:
No, we can remove those 👍🏻